tests/backup/backup_restore_basic_test.go

```diff
@@ -3189,3 +3190,145 @@ var _ = Describe("{DeleteNSDeleteClusterRestore}", func() {
+		sourceClusterUid         string
+		backupLocationMap        map[string]string
+		s3CloudCredName          string
+		s3BackupLocationName     string
+		s3CloudCredUID           string
+		s3BackupLocationUID      string
+		nfsBackupLocationName    string
+		nfsBackupLocationUID     string
+		bkpNamespaces            []string
+		labelSelectors           map[string]string
+		backupNames              []string
+		restoreNames             []string
+		numberOfAlternateBackups int = 2
+	)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("AlternateBackupBetweenNfsAndS3", "To perform alternate backups between NFS and S3, and then perform the restore", nil, 86088)
+		backupLocationMap = make(map[string]string)
+		labelSelectors = make(map[string]string)
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, appCtx := range appContexts {
+				appCtx.ReadinessTimeout = appReadinessTimeout
+				scheduledAppContexts = append(scheduledAppContexts, appCtx)
+				namespace := GetAppNamespace(appCtx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+
+	It("To validate alternate backups between Nfs And S3", func() {
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
+			ValidateApplications(scheduledAppContexts)
+		})
+
+		Step("Creating cloud setting for aws and backup locations for S3 and NFS", func() {
+			log.InfoD("Creating cloud setting for aws and backup locations for S3 and NFS")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			log.InfoD("Creating NFS backup location")
+			nfsBackupLocationName = fmt.Sprintf("%s-%s-bl-%v", "nfs", getGlobalBucketName("nfs"), RandomString(4))
+			nfsBackupLocationUID = uuid.New()
+			backupLocationMap[nfsBackupLocationUID] = nfsBackupLocationName
+			err = CreateNFSBackupLocation(nfsBackupLocationName, nfsBackupLocationUID, orgID, " ", getGlobalBucketName("nfs"), true)
+			dash.VerifyFatal(err, nil, "Creating backup location")
+			log.InfoD("Creating AWS cred and S3 backup location")
+			s3CloudCredName = fmt.Sprintf("%s-%s-%v", "cred", "s3", RandomString(4))
+			s3BackupLocationName = fmt.Sprintf("%s-%s-bl-%v", "s3", getGlobalBucketName("aws"), RandomString(4))
+			s3CloudCredUID = uuid.New()
+			s3BackupLocationUID = uuid.New()
+			backupLocationMap[s3BackupLocationUID] = s3BackupLocationName
+			err = CreateCloudCredential("aws", s3CloudCredName, s3CloudCredUID, orgID, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", s3CloudCredName, orgID, "aws"))
+			err = CreateS3BackupLocation(s3BackupLocationName, s3BackupLocationUID, s3CloudCredName, s3CloudCredUID, getGlobalBucketName("aws"), orgID, "")
```

Comment:
Add error verification step for CreateS3BackupLocation.

Reply:
Included the changes.